azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -642,6 +643,23 @@ public static void mergePodSpec(V1PodSpec podSpec, AzKubernetesV1PodTemplate pod
             .addVolumeMountsItem(volumeMountItem);
       }
     }
+
```

Comment:
Is it possible to take this part in another method? 
mergePodSpec (...) {
  mergeVolumes(..);
  mergeInitContainers(..);
  mergeVolumeMounts(..);
  mergeProbes(..);
}

Something like this?

Reply:
Good suggestion. I added a separate Utils class for this. Please review.